aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -165,4 +165,54 @@ RegisterOperators::~RegisterOperators() = default;
 RegisterOperators::RegisterOperators(RegisterOperators&&) noexcept = default;
 RegisterOperators& RegisterOperators::operator=(RegisterOperators&&) noexcept = default;
 
+
+CppFunction::CppFunction(KernelFunction func, std::unique_ptr<c10::FunctionSchema> schema)
+  : func_(std::move(func))
+  , schema_(std::move(schema))
+  {}
+
+Module::Module(const char* ns)
+  : ns_(ns)
+  {}
+
+Module::Module(Module&&) noexcept = default;
+Module& Module::operator=(Module&&) noexcept = default;
+
+
+Module&& Module::def(const char* schema) && {
+  register_.op(c10::RegisterOperators::options()
+    .schema(schema)
+    .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA));
+  return std::move(*this);
+}
+
+static std::string addNamespace(const char* ns, const char* unqual_name_or_schema) {
```

Comment:
Anonymous namespaces are more readable than the much overloaded static keyword and do the same thing here because this is not a method. You could also make it a static and private method on the module class if you're never calling it from anywhere else.